### ISSUE TITLE
Connect Travis CI

### ISSUE THREAD

**Issue by vitkon** [DeloitteDigitalUK/react-redux-starter-app#20]
surface issues early

**Comment by michael-martin** [comment]
@vitkon @mattdell So I reverted on the `npm run lint` question on the `start` task... :(

I spent a while fighting with the linter configuration locally today due to first the airbnb-config-base package not being updated for me locally when the rest of the packages were, and then for a while after I'd fixed things because I needed to re-open the project in Atom for it to pick up the changes.

Those are exactly the kind of issues that developers new to React shouldn't have to worry about. If that was happening to them, I'd rather they could get the app up and running and play with it, and then ask here for support.

However, in return, I promise I'll set up this CI this week or next so we still get properly enforced linting before anything gets into master! 

Does that seem reasonable?
